charts/headlamp/templates/ingress.yaml

```
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "headlamp.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
```

Latest stable version is released for `v1` so

```suggestion
apiVersion: networking.k8s.io/v1
```